source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml

```diff
@@ -18,123 +18,147 @@
     </div>
-    <script type="text/javascript">
-        var calificationPeriod = "@ViewBag.Period";
-        var calificationUserName = "@ViewBag.UserName";
-    </script>
+    <!-- /ko -->
+    <!--ko if: userView != 3 && !evaluation.readyForDevolution -->
+    <div class="top-right">
+      <h4>Comentarios Generales</h4>
+      <div class="comments-container">
+        <textarea class="content-editable big-comment" data-bind="value: generalComment, enable: isEvaluationEditable()"></textarea>
+      </div>
+    </div>
+    <!-- /ko -->
+  </div>
+  <table data-bind="attr: {class: numberOfColumns}, css: {table: true, 'calification-items': true}">
+    <thead id="header-fixed">
+      <tr>
+        <th class="calificators"><h1>Evaluadores</h1></th>
+        <!-- ko foreach: califications -->
+        <th class="calificators">
+          <span data-bind="if: hasShowIcon"><i data-tips="" data-bind="css: { icon: true, 'top-tip': true, 'hide-column': !show()}, click: $root.toggleVisibilityColumn"></i></span>
+          <span data-bind="text: evaluatorEmployee.replace('_company', 'Empresa')"></span>
+        </th>
+        <!-- /ko -->
+        <th class="calificators">
+          <p>BUTTON HERE</p>
+        </th>
+      </tr>
+      <tr>
+        <th class="title-evaluation">Calificación Final</th>
+        <!-- ko foreach: calificationsAverages -->
+        <th class="title-evaluation"><span data-bind="visible: show(), text: value || '-'"></span></th>
+        <!-- /ko -->
+        <th class="title-evaluation"></th>
+      </tr>
+    </thead>
+    <tbody data-bind="foreach: groups">
+      <tr>
+        <td class="sub-title-evaluation"><span data-bind="text: name"></span></td>
+        <!-- ko foreach: averages -->
+        <td class="sub-title-evaluation"><span data-bind="visible: show(), text: value || '-'"></span></td>
+        <!-- /ko -->
+        <td class="sub-title-evaluation"></td>
+      </tr>
+      <!-- ko foreach: items -->
+      <tr data-bind="css : {'row-selected' : isRowSelected()}">
+        <td><span data-bind="attr:{'data-tips': description}" class="top-tip rol wrap-text"><span data-bind="text: text"></span><i class="icon info"></i></span></td>
+        <!-- ko foreach: values -->
+        <td data-bind="css: { 'exceeds-expectations': showValue() && value() > 3 && value() <= 4 , 'poor': showValue() && value() < 3 && value() >= 1}">
+          <span data-bind="if: !editable"><span data-bind="text: value, visible: showValue()"></span></span>
+          <span data-bind="if: editable == true"><input class="calification-input" data-bind="value: value, css: { invalid: !isValid()}, event: { focusin: onFocus, blur: onBlur} " /></span>
+        </td>
+        <!-- /ko -->
+        <td>
+          <div data-bind="visible: hasComments ">
+            <input class="cmn-toggle-round" type="checkbox" data-bind="checked: showComments, attr: { id: key}" />
+            <label data-bind="attr: { for: key}"></label>
```

Comment:
If _label_ is really need and _id_ attribute is only used to make the relation between _label_ and _input_, there is a simpler and _legal_ alternative:

```
<label>
<input class="cmn-toggle-round" type="checkbox" data-bind="checked: showComments" />
</label>
```